### PR TITLE
Added single package return for latest_version, fixed other bug.

### PR DESCRIPTION
**Pull request by justinta**
Fixes these two issues on OpenSuse
https://jenkins.saltstack.com/view/2015.8%20Branch/job/salt-2015.8-linode-opensuse13/lastFailedBuild/

### CODE REVIEW

**Comment by cachedout** [comment]
Could you clarify which bug this is intended to fix? It's a little unclear from these changes what the intention is. Thanks.

**Comment by jtand** [comment]
@cachedout Oops, forgot to add those last night. Edited the original comment with the problems this fixes.

File: salt/modules/zypper.py [hunk 1]
```diff
@@ -162,8 +162,6 @@ def info_available(*names, **kwargs):
         if nfo.get("name"):
             name = nfo.pop("name")
             ret[name] = nfo
-        if nfo.get("status"):
-            nfo['status'] = nfo.get("status").split(" ")[0]
```
Comment (cachedout) [inline salt/modules/zypper.py]: Could you clarify why this needs to be removed completely?

**Comment by jtand** [comment]
@cachedout I added back in `nfo.get('status')`, but removed the split function. I'm not sure why the .split(" ")[0] was causing the issue. I'm not quite sure how to view the data that nfo dict provides. Any insight you have would be great! 

```python
if nfo.get('status'):
           nfo['status'] = nfo.get('status')
```

**Comment by jtand** [comment]
Go Go Jenkins!

**Pull request merged**